orttraining/orttraining/core/graph/pipeline_transformer.h

```diff
@@ -16,10 +16,22 @@ Status TransformGraphForPipeline(
     const std::unordered_set<std::string>& initializer_names_to_preserve,
     pipeline::PipelineTensorNames& pipeline_tensor_names);
 
-Status ApplyPipelinePartitionToMainGraph(
-    Graph& graph,
-    const std::vector<TrainingSession::TrainingConfiguration::CutInfo>& cut_info,
-    size_t pipeline_stage_id,
-    size_t num_pipeline_stage);
+// TODO(jufranc): when adapting this code to partition training graphs, add
+// a boolean is_training as parameter.
+Status ApplyPipelinePartitionToMainGraph(Graph& graph,
+    std::map<Node*, int>& op_to_stage,
+    int pipeline_stage_id,
+    int nstages);
+
+Status GetDeviceAssignmentMap(Graph& graph,
+                              const std::map<std::string, int>& id_to_stage,
+                              std::map<Node*, int>& op_to_stage,
+                              int nstages);
```

Comment:
nstages->num_stages.

Reply:
I believe I replaced all the occurrences of nstages with num_stages. Thanks.